src/components/dialogs/input-hooks.js

```diff
@@ -283,18 +284,30 @@ export const useConnectivityValue = ({
     }, [inputForm.toggleClear]);
 
     useEffect(() => {
-        setConnectivity({
-            voltageLevel: voltageLevelIdDefaultValue
-                ? voltageLevelOptions.find(
-                      (value) => value.id === voltageLevelIdDefaultValue
-                  )
-                : null,
-            busOrBusbarSection: busOrBusbarSectionIdDefaultValue
-                ? {
-                      id: busOrBusbarSectionIdDefaultValue,
-                  }
-                : null,
-        });
+        if (fetchedVoltageLevelOptions) {
```

Comment:
this seems interesting to test `fetchedVoltageLevelOptions` definition here.
Maybe prefer to do
`if(!fetchedVoltageLevelOptions) return;`
to avoid savage indentation in the following code (dedication @laurgarn )